specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/billingSavingsPlansList.json

```diff
@@ -0,0 +1,255 @@
+{
+  "parameters": {
+    "billingAccountName": "00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+    "api-version": "2023-04-01",
+    "$filter": "billingProfileId eq '/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/billingProfiles/AAAA-BBBB-CCC-DDD'",
+    "$orderby": "properties/displayName asc",
+    "refreshSummary": "true",
+    "selectedState": "Succeeded",
+    "take": "3"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/savingsPlanOrders/20000000-0000-0000-0000-000000000000/savingsPlans/30000000-0000-0000-0000-000000000000",
+            "name": "30000000-0000-0000-0000-000000000000",
+            "type": "microsoft.billing/billingAccounts/savingsPlanOrders/savingsPlans",
+            "sku": {
+              "name": "Compute_Savings_Plan"
+            },
+            "properties": {
+              "billingScopeId": "/subscriptions/10000000-0000-0000-0000-000000000000",
+              "billingAccountId": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+              "billingProfileId": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/billingProfiles/AAAA-BBBB-CCC-DDD",
+              "billingPlan": "P1M",
+              "commitment": {
+                "grain": "Hourly",
+                "currencyCode": "USD",
+                "amount": 10.0
+              },
+              "renew": false,
+              "appliedScopeType": "Shared",
+              "displayName": "SP1",
+              "provisioningState": "Succeeded",
+              "term": "P1Y",
+              "displayProvisioningState": "Succeeded",
+              "userFriendlyAppliedScopeType": "Shared",
```

Comment:
what's the difference between appliedScopeType and userFriendlyAppliedScopeType?

Reply:
Please refer to https://github.com/Azure/azure-rest-api-specs/blob/57f43d77fce5a2687282ff03d29bb4e128ad03e1/specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/savingsPlansListByBillingAccount.json#L205-L210

For Single-ResourceGroup case, on our backend we model it as "Single" scope, however we are displaying it as ResourceGroup in the UX so that it could be distinguished from "Single-Single" type(which is for a single subscription)

Reply:
That still can be done with one property right ? Just return the userFriendlyAppliedScope.

Reply:
We can take it offline discussion